src/ThemeProvider/index.tsx

```diff
@@ -44,4 +36,4 @@ const ThemeProvider: React.ComponentType<{ theme?: keyof typeof themes }> = ({
 };
 
 export default ThemeProvider;
-export const useTheme = () => React.useContext(ThemeContext as React.Context<typeof defaultTheme>);
+export { useTheme } from '@emotion/react';
```

Comment:
0% a criticism or complaint (as I get why this export is happening)... and yet, honestly, this pattern makes me uneasy.

Reply:
would love to discuss more :) are you thinking use a regular context we create?

Reply:
Not really. it's just the exporting a third party module what doesn't seem right to me.
Again, I know why it's needed, given Emotion's "special needs". Just thinking there may be other ways to have Emotion play along without this railguard enforcement.

Reply:
It could be cleaned up. I think it'll work just fine if `useTheme` is directly imported from `emotion` where ever it is needed.
This re-export was mostly a "don't break more than you need to" change.

Reply:
fair enough!!! "taking one day at a time" 👍  I like it.